examples/BME280/BME280.ino

```diff
@@ -1,40 +1,11 @@
-/*
-  WiFi Web Server LED control via web of things (e.g., WebThings Gateway)
-  based on WiFi101.h example "Provisioning_WiFiWebServer.ino"
-
- A simple web server that lets you control an LED via the web.
- This sketch will print the IP address of your WiFi device (once connected)
- to the Serial monitor. From there, you can open that address in a web browser
- to turn on and off the onboard LED.
-
- You can also connect via the Things Gateway http-on-off-wifi-adapter.
-
- If the IP address of your shield is yourAddress:
- http://yourAddress/H turns the LED on
- http://yourAddress/L turns it off
-
- This example is written for a network using WPA encryption. For
- WEP or WPA, change the Wifi.begin() call accordingly.
-
- Circuit:
- * WiFi using Microchip (Atmel) WINC1500
- * LED attached to pin 1 (onboard LED) for SAMW25
- * LED attached to pin 6 for MKR1000
-
- created 25 Nov 2012
- by Tom Igoe
-
- updates: dh, kg 2018
- */
-
```

Comment:
Why removing this header description? Can't you just update it?

Reply:
I am going to write a full markdown file on this.

Reply:
Do you want to do that in a subsequent PR? if so I'll add an issue after merging.

Reply:
Already created an issue for that. 
https://github.com/relu91/webthing-arduino/issues/9